src/app/utils/index.ts

```diff
@@ -30,13 +31,23 @@ export function getTheme() {
   });
 }
 
-export function getBrowserType(): BrowserType | null {
+export function useTheme(): Theme {
+  const { settings } = useSettings();
+
+  return settings.theme === "dark" ||
+    (settings.theme === "system" &&
+      window.matchMedia("(prefers-color-scheme: dark)").matches)
+    ? "dark"
+    : "light";
+}
+
+export function getBrowserType(defaultType?: BrowserType): BrowserType | null {
```

Comment:
I feel this param is a bit strange. I'd add the default inside the method (I see no use case where it would make sense to be able to configure this on a call level, I see it as a global default)

```
  // Default to "chrome"
  return "chrome";
```